portfolio/src/main/webapp/script.js

```diff
@@ -15,14 +15,12 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomImage() {
 
   // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+  var randNum = Math.floor(Math.random() * 7) + 1;
```

Comment:
It would be good to use "let" instead of "var" since "let" variable is block-scoped variable.